src/dataflow/core/linearize.py

```diff
@@ -96,6 +97,8 @@ def sexp_formatter(sexp_tokens: List[str]) -> List[str]:
 
 def sexp_to_seq(s: Sexp) -> List[str]:
     if isinstance(s, list):
+        if len(s) == 3 and s[0] == META_CHAR:
+            return [META_CHAR] + [y for x in [s[1], s[2]] for y in sexp_to_seq(x)]
```

Comment:
🐑  destructure `_meta_char, some_useful_name, some_other_useful_name = s`

Reply:
Sorry, I forgot to press push before I committed.